whitepaper/whitepaper.md

```diff
@@ -319,7 +318,9 @@ Automatic smart contract migration yields risks for enterprise blockchain system
 
 By expressing my preferences in terms of functional and non-functional requirements, Hyperledger Cactus can recommend a set of suitable blockchains, as the target of the migration.
 Firstly, I could know in real-time the characteristics of the target blockchain that would influence my decision.
-For instance, the platform can analyze see the cost of writing information to Ethereum, the exchange rate US dollar - Ether, the average time to mine a block, the transaction throughput, and the network hash rate ((Frauenthaler et al., 2019).
+
+For instance, the platform can analyze see the cost of writing information to Ethereum, the exchange rate US dollar - Ether, the average time to mine a block, the transaction throughput, and the network hash rate [3].
```

Comment:
Same as above regarding the reference link, example below:

`<sup>[3](#7-references)</sup>`

Reply:
Sure thing!